spec/circuits/var_len_keccak.md

```diff
@@ -63,13 +62,10 @@ We give the high-level description of the Keccak circuit, referring to component
 The circuit ID corresponding to a given $`vk`$ is $`\mathsf{C_{ID}} = \mathsf{keccak}(\mathsf{domain tag},vk)`$.
 
 #### Domain Tag
-The domain tag is derived either from the string `UPA Groth16 circuit id` (if the circuit does not use the LegoSNARK commitment) or else from the string `UPA Groth16 with commitment circuit id`. The domain tag is derived from this string as follows:
-1. Convert the domain tag string to bytes.
-2. Compute the keccak digest of those bytes, obtaining 32 bytes.
-3. Convert these 32 bytes to a field element, by considering them as the little-endian byte representation of a 256-bit integer and reducing modulo $`r`$.
+The domain tag is derived either from the string `UPA Groth16 circuit id` (if the circuit does not use the commitment to witness value) or else from the string `UPA Groth16 with commitment circuit id`. The string is interpreted as ASCII bytes and it's 32-byte Keccak digest is used as the corresponding domain tag.
 
 #### Byte Representation
-We encode $`vk`$ in bytes as $`\alpha || \beta || \gamma || \delta || \ell_s || s_0 || ... || s_{\ell_s-1}`$ if the circuit does not use the optional LegoSNARK commitment. If it does use the optional commitment we append $`h_1 || h_2`$ to the above bytes.
+We encode $`vk`$ in bytes as $`\alpha || \beta || \gamma || \delta || \ell_s || s_0 || ... || s_{\ell_s-1}`$ if the circuit does not use the optional commitment to witness value. If it does use the optional commitment we append $`h_1 || h_2`$ to the above bytes.
```

Comment:
has $\ell_s$ been defined before? I think this is what they were referring to in the telegram chat.

Reply:
I think \ell_s is defined, and they are just saying that $ell_{s,i}$ isn't used anywhere.

Reply:
($\ell_s$ is defined just below here)